### PR TITLE
Add forward kinematics demo for single finger

### PR DESCRIPTION
**Pull request by luator**
[//]: # "Thanks for your contribution.  To make life of the reviewers easier,"
[//]: # "please give this pull request a meaningful title and provide the"
[//]: # "requested information below."

## Description
Add a basic forward kinematics example using the Finger.

*Note:*  This is equivalent to https://github.com/open-dynamic-robot-initiative/blmc_robots/pull/52 but moved here as this is now the place for finger-related demos.

## How I Tested
By running it.

## I fulfilled the following requirements

[//]: # "Please make sure you followed these steps before requesting a review."
[//]: # "Check the boxes in the list below, when done."

- [x] All new code is formatted according to our style guide (for C++ run clang-format, for Python, run flake8 and fix all warnings).
- [x] All new functions/classes are documented and existing documentation is updated according to changes.
- [x] No commented code from testing/debugging is kept (unless there is a good reason to keep it).

### CODE REVIEW

**Comment by luator** [comment]
See discussion at the old PR: https://github.com/open-dynamic-robot-initiative/blmc_robots/pull/52

File: demos/demo_finger_forward_kinematics.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    action = robot.Action()
+    while True:
+        t = robot.frontend.append_desired_action(action)
+        joint_positions = robot.frontend.get_observation(t).position
+
+        # compute the forward kinematics
+        pinocchio.framesForwardKinematics(model, data, joint_positions)
+
+        # get the position of the tip link
+        pos = data.oMf[tip_link_id].translation
```
Comment (mwuethri) [inline demos/demo_finger_forward_kinematics.py]: are you sure this is the right frame transformation? i am not an expert with pinocchio, but i remember that @hammoudbilal explained me recently some slightly counterintuitive convention.

Comment (hammoudbilal) [reply demos/demo_finger_forward_kinematics.py]: @mwuethri  this computes the forward kinematics of all the frames and stores them in data.oMf , I think your confusion is regarding the jacobians

Comment (mwuethri) [reply demos/demo_finger_forward_kinematics.py]: right, then it's all fine.

**Pull request merged**